src/content/whats-new/2023/12/whats-new-12-21-errorsinbox-mobilegrouping.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Streamline error management with New Relic errors inbox mobile grouping'
+summary: 'Automatically group thousands of error events into common patterns for faster triage of mobile user issues'
+releaseDate: '2023-12-21'
+learnMoreLink: 'https://docs.newrelic.com/docs/errors-inbox/errors-inbox/#how-groups-work'
+---
+
+New Relic errors inbox now lets you automatically group mobile error events into common patterns, reducing noise, allowing you to focus on identifying and resolving the issues with the highest user impact before they hurt the business. With New Relic errors inbox mobile grouping now you can:
+* **Identify high-impact errors faster:** Automatically identify patterns across thousands of individual errors and turn them into meaningful groups, so you can zero in on the most important issue and priorities. 
+* **Analyze more errors with less effort:** Improve user experience for apps with high-volume mobile engagement while staying under cardinality limits so you can capture and examine the errors that matter most
```

Comment:
@ken-barrette would you consider splitting up this sentence like this?

```
* **Analyze more errors with less effort:** Improve the user experience for apps with high-volume mobile engagement while staying under cardinality limits. This allows you to capture and examine the errors that matter most.
```

Reply:
Yes, I will make that change.